backend/tests/test_integration_import.py

```
@@ -113,6 +114,28 @@ def test_import_creates_trades(self, client, mock_schwab):
         assert len(positions) == 2
         tickers = {p["ticker"] for p in positions}
         assert tickers == {"AAPL", "MSFT"}
+        # Both mock transactions are short-only legs (no shares ever
+        # acquired), so the derived label is "csp" for both — the
+        # AAPL sell_put has 0 shares + 1 open put, and the MSFT
+        # sell_call has 0 shares + 1 open call (anomaly path → "cc").
+        labels = {p["ticker"]: p["strategy"] for p in positions}
+        assert labels["AAPL"] == "csp"
+        assert labels["MSFT"] == "cc"
+
+    def test_import_accepts_legacy_strategy_field_but_ignores_it(self, client, mock_schwab):
```

Ruff (0.15.12)

[warning] Unused method argument: `mock_schwab`

(ARG002)